ai_commands/generate-pr-content.md

```
@@ -0,0 +1,118 @@
+Title: Generate PR Title and Description from Diff
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)